content/en/community/journey.md

```diff
@@ -0,0 +1,35 @@
+# Join the Journey: Shape the Future Together!  
```

Comment:
You will need to add the page headers specific to Hugo so that this page appears in the sidebar navigation. Please check other docs site pages for headers examples, something like:

```
---
title: "CHT Roadmap"
linkTitle: "Roadmap"
weight: 2
description: >
  CHT Initiatives
---
```